engines/query-sparql-file/package.json

```diff
@@ -172,10 +171,12 @@
     "@comunica/mediator-join-coefficients-fixed": "^2.6.8",
     "@comunica/mediator-number": "^2.6.8",
     "@comunica/mediator-race": "^2.6.8",
-    "@comunica/runner": "^2.6.8",
     "@comunica/runner-cli": "^2.6.8",
     "process": "^0.11.10"
   },
+  "devDependencies": {
+    "@comunica/types": "^2.6.8"
```

Comment:
Should be a normal dependency. (but is it used even?)

Reply:
It is used in `lib/QueryEngine.ts` does this not count?

Reply:
Typing packages should always be actual dependencies, otherwise users of comunica will see compilation errors due to missing types.